src/pypa/parser/parser.cc

```diff
@@ -1816,7 +1819,13 @@ bool expr_stmt(State & s, AstStmt & ast) {
                     case AstType::ListComp:
                         syntax_error(s, target, "can't assign to list comprehension");
                         return false;
-                    case AstType::Name:
+                    case AstType::Name: {
+                        auto name = std::static_pointer_cast<AstName>(target);
+                        if (disallowed_names.find(name->id) != disallowed_names.end()) {
```

Comment:
https://en.cppreference.com/w/cpp/container/unordered_set/contains

Reply:
```
external/com_github_vinzenz_libpypa/src/pypa/parser/parser.cc:1771:38: error: no member named 'contains' in 'std::unordered_set<std::basic_string<char>>'
                if (disallowed_names.contains(name->id)) {
                    ~~~~~~~~~~~~~~~~ ^
external/com_github_vinzenz_libpypa/src/pypa/parser/parser.cc:1830:46: error: no member named 'contains' in 'std::unordered_set<std::basic_string<char>>'
                        if (disallowed_names.contains(name->id)) {
                            ~~~~~~~~~~~~~~~~ ^
```

Reply:
<img width="196" alt="image" src="https://user-images.githubusercontent.com/5456019/180451678-d9d49091-1bd7-4ef2-a826-303a8d7a5e5b.png">

> Since C++20

Reply:
I don't think we have cpp 20 in Pixie either afaik.